server/package.json

```diff
@@ -38,10 +38,10 @@
     "passport-local": "^1.0.0",
     "passport-saml": "^1.1.0",
     "passport-strategy": "^1.0.0",
-    "pg": "^6.0.0",
+    "pg": "^6.4.2",
     "request": "^2.88.0",
     "request-promise-native": "^1.0.5",
-    "sequelize": "^5.15.1",
+    "sequelize": "^5.21.10",
```

Comment:
We shouldn't need to update the versions of any packages for this issue. It's most likely harmless, but for concision, clarity, and keeping within the scope of the issue, we should isolate the changes to just what's necessary. Please revert the changes to this file, and omit `package-lock.json` from the PR as well.

Reply:
@briandavidson - I fixed the problem with the allowable number of digits for the phone number field, I was able to successfully revert both the package.json and the package-lock.json files, but I'm not sure how to omit it from the PR. Can I just go in and delete it? I was worried that would remove it from Dev once we merge.

Reply:
@briandavidson - I was able to omit both .json files since there is no need to make any changes to them for this issue.